student.rb

```diff
@@ -6,6 +6,7 @@ class Student < Person
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
     @classroom = classroom
+    classroom.students << self
```

Comment:
- [ ] Logically in a single classrom there are going to be multiple students so here students object of the classroom is an array. I would suggest, instead of directly associating the  new student object with the classsrom.students array, only push the new student object into the classrom.student array to create the `has many` assocciation. Example snippet:

```ruby
 def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end
```